yarn-project/constants/src/scripts/constants.in.ts

```diff
@@ -156,6 +156,18 @@ const PIL_CONSTANTS = [
   'MAX_PACKED_PUBLIC_BYTECODE_SIZE_IN_FIELDS',
 ];
 
+const PIL_GENERATORS: string[] = [
```

Comment:
Generators weren't included in the PIL in the previous PR - followed the same approach as the `cpp` gen